.kokoro/tests/run_tests.sh

```diff
@@ -83,6 +83,9 @@ export PATH="${HOME}/.local/bin:${PATH}"
 # install nox for testing
 pip install --user -q nox
 
+# install PyYaml (used by the DRIFT region tag parsing system)
+pip install --user -q pyyaml
+
```

Comment:
If we introduce an envvar, can we only install pyyaml if the envvar is set to true? Also I think it is ok to install the parser alongside of pyyaml.

Or even, how about to have requirements.txt in the parser repo and run `pip install --user -r ${PARSER_DIR}/requirements.txt` or sth?

Reply:
I got this working by installing `pyyaml` in a specific directory and adding that to the `PYTHONPATH`. (I think there are some `virtualenv` issues here that prevent the usual "`pip install` and go" approach.)